modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -3,8 +3,8 @@ package controller
 import (
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	capabilitytypes "github.com/cosmos/ibc-go/modules/capability/types"
 
+	capabilitytypes "github.com/cosmos/ibc-go/modules/capability/types"
```

Comment:
super nit: I would consider external modules to be any module in the go.mod import, but this is fine with me as well